ols/app/endpoints/ols.py

```diff
@@ -96,7 +97,10 @@ def conversation_request(llm_request: LLMRequest) -> LLMResponse:
                 # See the comment in `summarize` method in `docs_summarizer.py`
                 # Because of that, we are ignoring some type checks when we
                 # are creating response model.
-                response = llm_response.response
+                if isinstance(llm_response, str):
+                    response = llm_response
+                else:
+                    response = llm_response.response  # type: ignore
```

Comment:
Can you check if the `# type: ignore` here (and bellow) is still needed after your changes?

Reply:
Yes, I need it because we could get string from langchain and Response from llama_index

Reply:
are you trying to access tuple item by it's name?

Reply:
Resolved it by checking for string/None